tools/PI/DevHome.PI/Controls/ExpandedViewControl.xaml

```diff
@@ -30,22 +30,32 @@
                     <RowDefinition Height="Auto"/>
                 </Grid.RowDefinitions>
 
-                <ListView
-                x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
-                 SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
-                    <ListView.ItemTemplate>
-                        <DataTemplate>
-                            <StackPanel Orientation="Horizontal">
-                                <TextBlock Text="{Binding IconText}" Style="{x:Null}"
-                                FontFamily="Segoe Fluent Icons" FontSize="12"
-                                HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                                <TextBlock Text="{Binding ContentText}"
-                                Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
-                            </StackPanel>
-                        </DataTemplate>
-                    </ListView.ItemTemplate>
-                </ListView>
-
+                <StackPanel Orientation="Vertical">
+                    <ListView
+                    x:Name="NavLinksList" SelectionMode="Single" VerticalAlignment="Stretch" ItemsSource="{x:Bind viewModel.Links}"
+                     SelectedIndex="{x:Bind viewModel.SelectedNavLinkIndex, Mode=TwoWay}" SelectionChanged="{x:Bind viewModel.Navigate}">
+                        <ListView.ItemTemplate>
+                            <DataTemplate>
+                                <StackPanel Orientation="Horizontal">
+                                    <TextBlock Text="{Binding IconText}" Style="{x:Null}"
+                                    FontFamily="Segoe Fluent Icons" FontSize="12"
+                                    HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                    <TextBlock Text="{Binding ContentText}"
+                                    Margin="12,0,0,0" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                                </StackPanel>
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <Rectangle x:Name="Separator"
+                        Visibility="{x:Bind viewModel.AppSettingsVisibility, Mode=OneWay}"

```

Comment:
XAML auto-converts from bool to Visibility. It's usually easier to make this a bool like IsAppSettingsVisible and use bools rather than Visibilities.

Reply:
Ahh... I had not realized that. Thanks!